### ISSUE TITLE
1.10.2 Update Request

### ISSUE THREAD

**Issue by AxolotlAccount** [Azanor/thaumcraft-suggestions#71]
Most of 1.7.10 and 1.8.9 mods are updating to 1.10.2. When will thaumcraft be updated to 1.10.2?

**Comment by angeltxilon** [comment]
I suppose that thaumcraft will delay to be updated.

Thaumcraft, unlike the most mods that are updated to 1.10.2, it is a very big mod, 
plus the battle and render mechanics of 1.9.X to upper versions are very different (this is, the reason why thaumcraft is not updated to 1.9.X)...
Yep, Azanor will delay to update the mod, due to the new mechanics of game and the new additions that he want to add.

Be patient and do not request more speed to the author, 
this is a thing that costs a lot to make and is practically free for you.

You should be grateful that this mod has not been abandoned like others big mods.

**Comment by Azanor** [comment]
Yup, I basically have to rewrite the mod from scratch which would normally take lots of time, but combined with that I currently only get a few hours a week to work on it due to RL commitments. Rest assured I am working on it - it is just extremely slow going.

**Comment by BSherb** [comment]
I'm honestly surprised this has gone on as long as it has. I remember my first runin with Thaumcraft back in the Yog Box. The fact that it's maintained the course under Azanor shows a lot of determination, and loyalty. A lot of us have faith this will continue for a long time. 

Keep up the great work Azanor!

**Comment by SpiderOverlord** [comment]
Hopefully the wait is well worth the reward, keep it up azanor this is in my opinion one of the best mods out there!

**Comment by Twisted-Code** [comment]
as others have noted, the dedication you've demonstrated is virtually unmatched by all but a few modders. the only names that come to mind are... you know, I can't even think of them. If anyone gets impatient with you in spite of all this, that's their loss. That said, seeing as there was something about focusing on 1.11 if you weren't finished by the time Forge 1.11 came out, I'm a little concerned as I can't really update my mod pack past 1.10. Is there any chance of you still releasing a 1.10 version? I'll even pitch in to help anyway way I can after I've finished learning to mod... though, mind you, I'm also planning on some sort of witchery 1.10 port, as it doesn't look like Emoniph is anywhere to be found.

**Comment by Sunconure11** [comment]
@macks2008 Emoniph's code is closed source and All Right's Reserved, make your own mod similar to Witchery, don't do a direct port as you will have legal shenanigans.

**Comment by Somedudesbutonlyone** [comment]
Add More Taint! Taint is just a beast to slay, then It reappears then to slay again and again and AGAIN I recommend taint-ey tum tome. I know work on thaumcraft is hard But, HUIAU I love taint and hate taint.
